docs/advanced_topics/customisation/page_editing_interface.md

```diff
@@ -90,6 +90,8 @@ The process for creating new features is described in the following pages:
 -   [](../../extending/rich_text_internals)
 -   [](../../extending/extending_draftail)
 
+You can also provide a setting for naming a group of rich text features. See [](../../reference/settings.html#id17).
```

Comment:
The anchor `#id17` is autogenerated and likely to move about as new items are added / removed. I'd suggest using the named anchor, as we do here:

https://github.com/wagtail/wagtail/blob/86c3dbb961959c35252997034c1a5354711db9b6/docs/extending/rich_text_internals.md?plain=1#L202

Reply:
> The anchor `#id17` is autogenerated and likely to move about as new items are added / removed. I'd suggest using the named anchor, as we do here:
> 
> https://github.com/wagtail/wagtail/blob/86c3dbb961959c35252997034c1a5354711db9b6/docs/extending/rich_text_internals.md?plain=1#L202

Got it!